src/styles/main.scss

```diff
@@ -1,14 +1,6 @@
-html {
-  font-family: Roboto, sans-serif;
-}
+@import "./utils/variables";
 
-body {
-  margin: 0;
-}
-
-* {
-  box-sizing: border-box;
-}
-
-@import "_header.scss";
-@import "_catalog.scss";
+@import "./blocks/normalize.scss";
+@import "./blocks/header.scss";
+@import "./blocks/nav.scss";
+@import "./blocks/catalog.scss";
```

Comment:
This imports  would be better to be at the beginning of the page, but at the end